docs/Type-Providers.md

```diff
@@ -77,3 +84,7 @@ server.get('/route', {
     const { foo, bar } = request.query // type safe!
 })
 ```
+
+TypeBox uses the properties `kind` and `modifier` internally. These properties are not strictly valid JSON schema which will cause AJV to throw an invalid schema error. To remove the error it's either necessary to omit the properties by using [`Type.Strict()`](https://github.com/sinclairzx81/typebox#strict) or use the AJV options for adding custom keywords.
```

Comment:
```suggestion
TypeBox uses the properties `kind` and `modifier` internally. These properties are not strictly valid JSON schema which will cause `AJV@7` to throw an invalid schema error. To remove the error it's either necessary to omit the properties by using [`Type.Strict()`](https://github.com/sinclairzx81/typebox#strict) or use the AJV options for adding custom keywords.
```

Reply:
Sounds good! Perhaps it should be something like AJV@7 and newer, because it looks like fastify 4 will come with AJV@8 ?

Reply:
Yes